core/src/main/java/org/apache/accumulo/core/client/sample/AbstractHashSampler.java

```diff
@@ -77,13 +74,6 @@ public void init(SamplerConfiguration config) {
     String hasherOpt = config.getOptions().get("hasher");
     String modulusOpt = config.getOptions().get("modulus");
 
-    requireNonNull(hasherOpt, "Hasher not specified");
-    requireNonNull(modulusOpt, "Modulus not specified");
```

Comment:
This was removed, but was not replaced with any kind of validation that the required options were set with actual values.

Reply:
My recent commit moved the null check to where the other argument is checked.

Reply:
Not quite. What you added was a check that the key was not null, which might be redundant, since the fact that it exists implies it is not null. This code here was checking that the values (not the keys) that correspond to these option keys was not null.

Reply:
Ah, you are correct. I misread that. My future changes should handle this.